src/resolvers/message.ts

```diff
@@ -1,25 +1,76 @@
-import { Message, Resolvers } from '../generated/graphql';
+import { Message, MessagesConnection, Resolvers } from '../generated/graphql';
+import { Op, WhereOptions } from 'sequelize';
 
 import { checkAuth } from '../utils/auth';
+import paginateResults from '../utils/pagination';
 
 const resolver: Resolvers = {
   Query: {
     messages: async (
       _,
-      args, {
-        verifyUser,
-        models,
-      },
-    ): Promise<Message[]> => {
+      args,
+      { verifyUser, models },
+    ): Promise<MessagesConnection> => {
       const { Message: messageModel } = models;
       const auth = verifyUser();
       checkAuth(auth);
 
-      return messageModel.findAll({
-        where: {
-          messageId: 1,
-        },
+      const { message, channelId, pageSize = 20, after } = args;
+      let qryMsg: object = {};
+      let qryAfter: object = {};
+      let qryChannelId: object = {};
+
+      if (message) {
+        const msgOpConds: object = {};
+        const findColName = 'text';
+        msgOpConds[findColName] = {
+          [Op.like]: `%${message}%`,
+        };
+        qryMsg = {
+          [Op.or]: msgOpConds,
+        };
+      }
+      if (after) {
+        qryAfter = {
+          createdAt: { [Op.lt]: Number(after) },
+        };
```

Comment:
I don't think this style is good for readability. Similar things keep happening resulting in more lines of code.

Reply:
I referred to [following code](https://github.com/dooboolab/hackatalk-server/blob/1ce4941a2dfd3cbe7344f0e4852267160ceac168/src/resolvers/user.ts#L110) and did like that style. you want it to put in `where` attributes?

I will consider what is the good code style for readability, but please let me know something like what is prefer code style like sequelize's condition should be in the bracket or etc.

Reply:
It is not a big issue if you can find out easily like that. However, if more things come over and we need to make a massive conditional `if` statement, I think it isn't a good idea to just arrange them out. We need a discussion or you can try yourself first and propose your approach to us.